<ui version="4.0">
 <class>TNMBuilderDialogBase</class>
 <widget class="QDialog" name="TNMBuilderDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TNM Builder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Roads:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="roadLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_receivers">
     <property name="text">
      <string>Receivers:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="receiversLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_buildings">
     <property name="text">
      <string>Buildings:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="buildingsLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="label_existing_barriers">
     <property name="text">
      <string>Existing Barriers:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="existingBarriersLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>


   <item>
    <widget class="QLabel" name="label_elevation_contours">
     <property name="text">
      <string>Elevation Contours:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="elevationContoursLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_terrain_lines">
     <property name="text">
      <string>Terrain Lines:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="terrainLinesLayerComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Existing TNM .txf File:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="existingFileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseExistingButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_output">
     <property name="text">
      <string>Output TNM .txf File:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_output">
     <item>
      <widget class="QLineEdit" name="outputFileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseOutputButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="updateFileButton">
     <property name="text">
      <string>Update TNM .txf File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
